### ISSUE TITLE
Validator failing when both "graphics-document" and "document" specified in SVG role - reopen

### ISSUE THREAD

**Issue by KDean-Dolphin** [validator/validator#2037]
### Summary

This is a reopen of #1987.

The presence of both "graphics-document" and "document" is recommended in the ARIA specification. See the NOTE in this section:

https://www.w3.org/TR/graphics-aria-1.0/#graphics-document

### URL for document that reproduces the problem

n/a

### HTML/CSS/SVG snippet that reproduces the problem

```html
See #1987.
```

### Checker error messages

```shell
Discarding superfluous token “document” from value of attribute “role”. Browsers only process the first token found that is a defined ARIA non-abstract role.
```

### Spec links/citations

See #1987.

### Steps to reproduce

See #1987.

### Expected behavior

See #1987.

### Actual behavior

See #1987.

**Comment by sideshowbarker** [comment]
Fixed in https://github.com/validator/validator/commit/6c5c68acb13b15974bd65a45aa984ae15b7b4e3c and deployed:

https://validator.w3.org/nu/?showsource=yes&doc=data%3Atext%2Fhtml%3Bcharset%3Dutf-8%3Bbase64%2CPCFET0NUWVBFIGh0bWw%2BPGh0bWwgbGFuZz0iIj48bWV0YSBjaGFyc2V0PXV0Zi04Pg0KPHRpdGxlPlRlc3Q8L3RpdGxlPg0KPHN2ZyBpZD0iZXhwb3J0LXN2ZyIgd2lkdGg9IjEwMCUiIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyIgY2xhc3M9ImZsb3djaGFydCIgc3R5bGU9Im1heC13aWR0aDogMzQ4LjM5MDYyNXB4OyBiYWNrZ3JvdW5kOiByZ2IoMjU1LCAyNTUsIDI1NSk7IiB2aWV3Qm94PSIwIDAgMzQ4LjM5MDYyNSA0OTEuNzY1NjI1IiByb2xlPSJncmFwaGljcy1kb2N1bWVudCBkb2N1bWVudCIgYXJpYS1yb2xlZGVzY3JpcHRpb249ImZsb3djaGFydC12MiI%2BPC9zdmc%2B

**Comment by KDean-Dolphin** [comment]
@sideshowbarker Still a problem. This is the latest output, showing that it is picking up the recently deployed vnc-jar. If you want the HTML, email me at the address in my GitHub profile.

  $ java -jar "/home/runner/work/_actions/w3c/spec-prod/v2/node_modules/.pnpm/vnu-jar@26.2.5/node_modules/vnu-jar/build/dist/vnu.jar" --also-check-css index.html
    "file:/home/runner/work/vc-use-cases/vc-use-cases.w3c/index.html":147.1-147.31: info warning: The “type” attribute is unnecessary for JavaScript resources.
    "file:/home/runner/work/vc-use-cases/vc-use-cases.w3c/index.html":696.76-696.279: error: Discarding superfluous token “document” from value of attribute “role”. Browsers only process the first token found that is a defined ARIA non-abstract role.
    "file:/home/runner/work/vc-use-cases/vc-use-cases.w3c/index.html":698.10090-698.10114: error: Element “foreignObject” is missing required attribute “height”.
